### PR TITLE
Frontend-Backend Initial Communication

### PR DESCRIPTION
**Pull request by JesseEmond**
This sets up the project to communicate with the REST API. It now `GET`s to `/foo` and shows the message returned from the API.

Notable *project* changes:
- [Volley](https://developer.android.com/training/volley/index.html) library added to the project
- `developer-config.gradle` for developer-specific configuration
- I took on the opportunity to change `buildToolsVersion '22.0.1'` as well, hoping that this will simplify our members' lives having to deal with *AndroidStudio*

Peer-programmed with @MicLamb

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -30,3 +32,27 @@ pour voir quelque chose au debut)
...
+
+To figure out the IP address on your computer, you must do the following in a
+command prompt:
+- `ipconfig` on Windows
+- `ip addr` or `ifconfig` on Linux
+
+You can then go in your `app/developer-config.gradle` config to update your server
+endpoint accordingly (follow the documentation in the file).
+
+**Note**: your device and server must be able to communicate together. For example,
```
Comment (DrPandemic) [inline frontend/README.md]: Would be interresting to talk about firewalls for the real devices. It could cause some problem

**Comment by Erikad94** [comment]
:1234:  :+1:  :8ball: :100:

**Comment by MicLamb** [comment]
MicLamb approves this.

**Comment by JesseEmond** [comment]
@DrPandemic: I added a comment about firewalls (40a20273b44291a0a34eb086962c7275b95fcddf). When you have time, please take a look and merge if you are satisfied with the PR.

**Comment by DrPandemic** [comment]
:+1: but shouldn't we do unit testing?

**Pull request merged**